.vscode/settings.json

```diff
@@ -23,7 +23,7 @@
     "**/lib/**/*.js": true,
     "**/node_modules": false
   },
-  "editor.formatOnSave": true,
```

Comment:
>true [](http://example.com/codeflow?start=25&length=4)

Is it an intentional change?

Reply:
Yes. This makes VS Code break formatting in C++ files.